### ISSUE TITLE
Remove Firefox Accounts

### ISSUE THREAD

**Issue by ckprice** [mozilla/testpilot#1035]
- [ ] Remove FxA connections.
- [ ] Remove django-allauth from project (which enables #1034).
- [ ] Remove user email and "Logout" from `[...]` menu.
- [ ] Remove Firefox Accounts oauth credentials.

**Comment by lmorchard** [comment]
For what it's worth, this bug looked "too large" to me, so I split it up a bit and added some more issues - #1040, #1041, #1042, #1043. Will probably do some more tweaking as I think through what else is touched by FxA and user accounts, because there are some weird corners

**Comment by jaredhirsch** [comment]
Do we also want to ask the FxA team to deactivate our oauth credentials? Not sure if this should be a checkbox or a separate coordination TODO.

**Comment by lmorchard** [comment]
Seems like #1047 is covering most of the visible & usable aspects of this issue. So, I'm considering this one as the final cleanup steps - e.g. remove django-allauth from the project, delete the contents of the socialaccounts tables, clean up anything else lingering related to FxA short of wiping the users (#1034)

**Comment by lmorchard** [comment]
Something occurs to me: Once #1065 is merged, we won't actually have Firefox Accounts wired up on the server side anymore. So, this issue _could_ possibly lag behind the deployment of that stuff along with the user database cleanup.
